src/widgets/histogram/content-view.js

```diff
@@ -93,6 +93,10 @@ module.exports = cdb.core.View.extend({
       return;
     }
 
+    if (this._dataviewModel.changed.bins) {
+      this.binsCount = this._dataviewModel.get('bins');
```

Comment:
Why are we storing a private var instead of just using `this._dataviewModel.get('bins')` where it's needed?